evm_arithmetization/src/all_stark.rs

```diff
@@ -123,6 +123,10 @@ pub(crate) fn all_cross_table_lookups<F: Field>() -> Vec<CrossTableLookup<F>> {
     ]
 }
 
+/// Position of the Memory CTL within the vector of CTLs returned by
+/// `all_cross_table_lookups()`.
+pub(crate) const MEMORY_CTL_INDEX: usize = Table::Memory as usize;
```

Comment:
Is it temporary? The fact that they match is a coincidence, and isn't very reliable.

Reply:
Writing a hardcoded number also isn't reliable long term. For now it works because we have 2 CTLs for Keccak

Reply:
If this is to change, this would get caught anyway because of MemoryCTL failing, so not sure this is a big deal. We could also revamp the CTLs through some kind of `Enum` access similarly to the `Table` stuff, but I'd rather limit the change scope of release PRs.